tests/acceptance/blocks/facetscomponent.js

```diff
@@ -0,0 +1,48 @@
+import FilterBoxComponentBlock from './filterboxcomponent';
+
+/**
+ * This class models user interactions with the {@link FacetsComponent}.
+ */
+export default class FacetsComponentBlock {
+  constructor () {
+    this._filterBox = new FilterBoxComponentBlock('.js-yxt-Facets');
+  }
+
+  /**
+   * Return the child filter box component's block.
+   */
+  async getFilterBox () {
```

Comment:
Do all these methods need "async" qualifiers? I don't think this one does at least.

Reply:
Yeah I wasn't very clear on how these Selector promises (?) were working (still a little foggy). My current understanding is declaring a selector doesn't need async but using any of its methods that inspect the dom does?